apps/koala-frontend/src/app/core/pages/about/about.page.html

```diff
@@ -1 +1,57 @@
-About
+<h1>Über Koala</h1>
+
+<p>
+  Koala ist eine Entwicklung der Hochschule für Musik Freiburg und der Hochschule für Musik und Darstellende Kunst
+  Stuttgart. Die Entwicklung der App wurde von der
+  <a href="https://stiftung-hochschullehre.de/" target="_blank">Stiftung Innovation in der Hochschullehre</a> gefördert.
+</p>
+<br />
+<div class="card">
```

Comment:
instead of class="card" and then using the panel, you could also directly use <p-card> replacing the div and the panel (same result, but more direct definition what the goal is).

Reply:
Alright! Just copied the primeng example :)